src/index.ts

```diff
@@ -170,6 +170,36 @@ const main = async () => {
     res.json(data);
   });
 
+  app.get("/text-stories/search/:cursor/:searchInput", async (req, res) => {
+    let cursor = 0;
+    const nCursor = parseInt(req.params.cursor);
+    if (!Number.isNaN(nCursor)) {
+      cursor = nCursor;
+    }
+
+    let searchInput = req.params.searchInput;
+    const limit = 21;
+    const stories = await getConnection().query(`
+      select
+      ts.id,
+      u.username "creatorUsername",
+      u."photoUrl" "creatorAvatarUrl",
+      u.flair
+      from text_story ts
+      inner join "user" u on u.id = ts."creatorId"
+      where u.username ILIKE %$1%
```

Comment:
Hey, I tried making a request but the `%$1%` doesn't work for me. I think it's because of the missing apostrophes. Did you try it? @FluentCoding